packages/client/middleware/timeDelay/src/index.ts

```diff
@@ -131,4 +131,9 @@ export class TimeDelayMiddleware
 
     return { instructions, signers: [] };
   }
+
+  public async onClose(): Promise<MiddlewareResult> {
+    // TODO: implement
```

Comment:
Does this need to be done?

Reply:
Yes, but not tonight. It's not part of the audit-related program code. This specific middleware just needs to clean up it's state.